src/ape/managers/converters.py

```diff
@@ -140,6 +141,29 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 list_tuple_converter = ListTupleConverter(None, None, None)  # type: ignore
 
 
+class TimestampConverter(ConverterAPI):
+    """
+    Converts string of %m-%d-%Y %H:%M:%S format to a timestamp
+    """
+
+    def is_convertible(self, value: str) -> bool:
+        if not isinstance(value, str):
+            return False
+        if " " not in value or len(value.split(" ")) > 2:
```

Comment:
Check this out:  https://dateparser.readthedocs.io/en/latest/

^ Not saying we need to use this right now... but this seems like a tool that does what we would ideally like, we can parse any str to a `datetime` object that `dateparser` supports

Reply:
The problem with this is it does run into some issues (sometimes 11/12 will be recorded as November 12th, and sometimes its recorded as December 11th). Not a frequent issue, but when making smart contracts, I think this has to be more reliable?